Podfile

```diff
@@ -0,0 +1,12 @@
+platform :ios, '9.3'
```

Comment:
I think Eureka was integrated using Carthage, that is why the Podfile wasn't there

Reply:
Several other Eureka Pods (such as SuggestionRow) support cocoapods, carthage and Apple's dependency manager. The cocoapod files weren't supposed to be removed when support for carthage wad added.

Reply:
ImageRow supports Carthage and Cocoapods. It just needs the podspec to support Cocoapods (not the Podfile). The Podfile is used to install the dependencies for the example project, in this case Eureka. If I remember correctly, that has always been done using Carthage instead of Cocoapods in this project (ImageRow). We can use either of them but we should choose one and the instructions in the Readme tell the user to use Carthage to run the Example project. I don't see a reason for switching to Cocoapods. If the integration wasn't working for you then maybe we should improve the Readme